data/philly-bike/stations.csv

```diff
@@ -0,0 +1,141 @@
+,name,lat,lon,t0_firsttrip,t0_lasttrip,nrides
```

Comment:
missing `id` ?
why `t0_` and `nrides` ?  Let's make keys consistent in order to work in the viz?